### PR TITLE
Fixing the EAV selected scope for the generated attributes

### PR DESCRIPTION
**Pull request by eduard13**
**Description** (*)
This PR aims to:
- Fix the selected **SCOPE** attribute for generated EAV attributes
- Adding the returns for apply methods

**Fixed Issues (if relevant)**
N/A

**Questions or comments**

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft [hunk 1]
```diff
@@ -58,6 +58,7 @@ class ${CLASS_NAME} implements ${IMPLEMENTS} {
             #end
 ]
         );
+        return $this;
```
Comment (VitaliyBoyko) [inline resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft:61]: But this is completely useless. See `vendor/magento/framework/Setup/Patch/PatchApplier.php:157`. I think the issue related rather to the wrong doc block.
Moreover, I see no patches in the core implemented in such a way.

Comment (eduard13) [reply resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft:61]: Hi @VitaliyBoyko, totally agree with you 💯. But checking the `\Magento\Framework\Setup\Patch\PatchInterface::apply`, we may see the `return` annotation.

> Moreover, I see no patches in the core implemented in such a way.

Unfortunately, there are many places like that:
- \Magento\Analytics\Setup\Patch\Data\ActivateDataCollection::apply
- \Magento\Analytics\Setup\Patch\Data\PrepareInitialConfig::apply
- \Magento\Customer\Setup\Patch\Data\MigrateStoresAllowedCountriesToWebsite::apply

But I agree that there is no reason of it.

Comment (VitaliyBoyko) [reply resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft:61]: I think that annotation is wrong. 😑 We do not need to support it in the plugin

Comment (eduard13) [reply resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft]: Yeap 👍, removed it

**Review by VitaliyBoyko — APPROVED** [review]

**Pull request merged**